lib/raidenclient/types.ts

```diff
@@ -68,6 +68,7 @@ export type RaidenResolveRequest = {
   reveal_timeout: number;
```

Comment:
Should we be removing reveal and settle timeout as part of this PR? Or are you planning to do this separately? I'm thinking if it's done together this can be marked as closing #1149.

If so, you can also remove the commented out "unused" fields below.

Reply:
Yes, absolutely - good idea.